mixnet/util/src/lib.rs

```diff
@@ -0,0 +1,42 @@
+use std::{collections::HashMap, net::SocketAddr, sync::Arc};
+
+use parking_lot::Mutex;
+use tokio::net::TcpStream;
+
+#[derive(Clone)]
+#[repr(transparent)]
```

Comment:
Ah, I get used to mark a single field struct with `#[repr(transparent)]`. But I do not have a deep understanding of the pros and cons.

Reply:
It tells the compiler that you want the in-memory representation of the struct to be exactly the same as the only field. It's only useful if you need to inspect raw representation or do FFI and it could actually keep the compiler from doing optimizations (or you from adding a new field without having to remove it). 

Unless we have a clear reason to have this, let the compiler figure out what's best

Reply:
Thanks! 👍